airflow/providers/amazon/aws/hooks/secrets_manager.py

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import base64
+import json
+from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+
+
+class SecretsManagerHook(AwsBaseHook):
+    """
+    Interact with Amazon SecretsManager Service.
+
+    Additional arguments (such as ``aws_conn_id``) may be specified and
+    are passed down to the underlying AwsBaseHook.
+
+    .. see also::
+        :class:`~airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(client_type='secretsmanager', *args, **kwargs)
+
+    def get_secrets(self, secret_name: str) -> dict:
```

Comment:
might be better to just return the raw secret bytes here and let caller do the json.loads manually instead. there is no guarantee stored secret string will be a valid json object, especially for SecretBinary.

Reply:
yes, returning raw secrets now

Reply:
@houqp In which cases a secret would not be a valid json object?

Reply:
@houqp In which cases a secret won't be a valid json object?

Reply:
we store a lot of none json secret values in secret managers, for example, simple database passwords without the quote, private certificate keys, etc. You can store raw binary secret as well, which by definition won't be in json format.

Reply:
What about something like this for string secrets (not binary):
```
try:
    secret = json.loads(secret)
except JSONDecodeError:
    pass
```
That way if its a JSON we get directly the dict.

Reply:
we just need to add a debug log before the "pass"

Reply:
An idea what @houqp suggested and I follow is to make it as low level as possible and do not do preprocessing so caller always have flexibility what to do with Secrets. If we need secrets as a dict I would add wrapper on top of existing function that returns `str` that'll return dict. What about this?

Reply:
yeah, it would be much better to add a special wrapper for it. For secrets that the caller is sure to not be stored in JSON format, there is no point incurring this extra doomed to fail json decoding at runtime.

Reply:
Added one more function to return secrets as dict